src/main/java/net/openhft/chronicle/core/internal/analytics/ReflectionUtil.java

```diff
@@ -46,7 +46,7 @@ public static Method methodOrThrow(@NotNull final String className,
             return analyticsClass.getMethod(methodName, parameterTypes);
         } catch (ReflectiveOperationException e) {
             Jvm.rethrow(e);
-            return null;
+            throw new IllegalStateException("We cannot end up here");
```

Comment:
again it would be a bit nicer on static analysis if we could `throw Jvm.toUnchecked(e)`